engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-errata-modal.controller.js

```diff
@@ -68,7 +65,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkErrataModalC
         if ($scope.allHostsSelected) {
             $scope.errataActionFormValues.scopedSearch = hostIds.included.search;
         } else if (hostIds.included.ids.length > 0) {
-            $scope.errataActionFormValues.hostIds = hostIds.included.ids.join(',');
+            $scope.errataActionFormValues.hostIds = angular.toJson(hostIds.included.ids);
```

Comment:
we want to include the entire 'bulk' structure, so here it hink you'd just want:
`$scope.errataActionFormValues.hostIds = angular.toJson(hostIds)`

Reply:
also, there's a couple more UI controllers that will need an update: 

$ grep -R hostIds engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/ | grep join
engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-module-streams-modal.controller.js:            $scope.moduleStreamActionFormValues.hostIds = hostIds.included.ids.join(',');
engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-packages-modal.controller.js:                $scope.packageActionFormValues.hostIds = selectedHosts.included.ids.join(',');

Reply:
Ok, I'll update those as well.

Side note: the remote execution controller seems to be very light in the way of tests. I may add a few there too.